File: wrappers/java/src/main/java/com/evernym/vdrtools/pool/PoolLedgerTerminatedException.java
package com.evernym.vdrtools.pool;

import com.evernym.vdrtools.ErrorCode;
import com.evernym.vdrtools.IndyException;

/**
 * Exception thrown when a pool ledger has been terminated.
 */
public class PoolLedgerTerminatedException extends IndyException
{
	private static final long serialVersionUID = 768482152424714514L;
	private final static String message = "The pool ledger was terminated.";

	/**
	 * Initializes a new PoolLedgerTerminatedException.
	 */
	public PoolLedgerTerminatedException()
	{
		super(message, ErrorCode.PoolLedgerTerminated.value());
	}
}

File: wrappers/dotnet/indy-sdk-dotnet/PoolApi/PoolLedgerTerminatedException.cs
namespace Hyperledger.Indy.PoolApi
{
    /// <summary>
    /// Exception thrown when the pool ledger was terminated.
    /// </summary>
    public class PoolLedgerTerminatedException : IndyException
    {
        /// <summary>
        /// Initializes a new PoolLedgerTerminatedException.
        /// </summary>
        internal PoolLedgerTerminatedException(string message) : base(message, (int)ErrorCode.PoolLedgerTerminated)
        {

        }
    }

}
